.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "npm"
```

Comment:
Does this work with `pnpm`? I couldn't find a reference in the docs. (https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem)

Reply:
It's not yet supported: https://github.com/dependabot/dependabot-core/issues/1736

Reply:
Maybe just use it for GitHub actions for now?

Reply:
@Uzlopak

Reply:
implement pnpm in dependabot-core

Reply:
nevermind. I will not start learning ruby.

I recommend to use npm with package-lock=false

Reply:
Can you remove pnpm lock and start using with package lock false?